src/transformers/models/whisper/modeling_flax_whisper.py

```diff
@@ -58,6 +59,20 @@
 remat = nn_partitioning.remat
 
 
+def sinusoidal_embedding_init(key, shape, dtype=jnp.float_) -> jax.Array:
+    """Returns sinusoids for positional embedding"""
+    length, channels = shape
+    if channels % 2 != 0:
+        raise ValueError(
+            f"Number of channels has to be divisible by 2 for sinusoidal positional embeddings, got {channels} channels."
+        )
+    max_timescale = 10000
+    log_timescale_increment = math.log(max_timescale) / (channels // 2 - 1)
```

Comment:
(nit) maybe cleaner to define it on one line
```suggestion
    log_timescale_increment = math.log(10000) / (channels // 2 - 1)
```